provider/internal/java/dependency_test.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/go-logr/logr/testr"
 	"github.com/konveyor/analyzer-lsp/provider"
 )
+// createDepLabelItem creates a label item with the provided key-value pair.
+func createLabelItem(key, value string) string {
```

Comment:
no need to define another function in the test, should be able to use exported function from labels package